tests/test_processing.py

```diff
@@ -1,10 +1,10 @@
 """Tests for processing"""
 import numpy as np
 import pytest
-import test_processing_data as xtd
 import xarray as xr
 
 from scores.processing import broadcast_and_match_nan
+from tests import test_processing_data as xtd
```

Comment:
I think these module names are potentially confusign to pytest test discovery mechanisms.

Reply:
What if we use synthetic_ as an alternative?